### ISSUE TITLE
Technical Debt: Remove Pinned Gradle Dependency Version

### ISSUE THREAD

**Issue by luelista** [MaibornWolff/codecharta#2845]
org.eclipse.equinox.preferences ver. 3.10.0, a dependency of org.sonarsource.java, references a non-existent dependency ("org.osgi.service:org.osgi.service.prefs", but should be "org.osgi:org.osgi.service.prefs").

https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.equinox.preferences/3.10.0/org.eclipse.equinox.preferences-3.10.0.pom

Therefore we pinned the latest working version 3.9.100

https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.equinox.preferences/3.9.100/org.eclipse.equinox.preferences-3.9.100.pom

```
    implementation('org.eclipse.platform:org.eclipse.equinox.preferences') {
        version {
            strictly '3.9.100'
        }
    }
```

This should be removed as soon as a fixed org.eclipse.equinox.preferences is released - check here: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.equinox.preferences/

**Comment by ce-bo** [comment]
A new patch version is published. So I remove the blocked label and we can check if it is fixed in 3.10.1.

**Comment by phanlezz** [comment]
@BridgeAR Has this issue been resolved by PR #3004 ?

**Comment by BridgeAR** [comment]
Resolved by #3004
